cpp/foxglove/include/foxglove/server.hpp

```diff
@@ -216,12 +216,20 @@ struct WebSocketServerOptions {
   std::vector<std::string> supported_encodings;
   /// @brief A fetch asset handler callback.
   FetchAssetHandler fetch_asset;
+  /// @brief A sink channel filter callback.
+  SinkChannelFilterFn sink_channel_filter;
+  /// @cond foxglove_internal
```

Comment:
Did we want to make `tls_identity` foxglove internal? I thought this was shipped. It's in the documentation, at least?